pkg/proxy/ipvs/proxier.go

```diff
@@ -430,11 +430,6 @@ func NewProxier(ipt utiliptables.Interface,
 
 	klog.V(2).Infof("nodeIP: %v, isIPv6: %v", nodeIP, isIPv6)
 
-	if len(scheduler) == 0 {
```

Comment:
we want to keep this check actually, since scheduler is passed into Proxier still